src/main/java/xyz/nucleoid/plasmid/game/GameWorld.java

```diff
@@ -170,6 +265,11 @@ public JoinResult offerPlayer(ServerPlayerEntity player) {
         }
     }
 
+    /**
+     * Closes this {@link GameWorld} if it is not already closed.
+     *
+     * <p>Upon close, all players in this {@link GameWorld} are removed (kick implementation is up to the {@link BubbleWorld} associated with this {@link GameWorld}).
```

Comment:
Without the mention of bubble world, could just say they are kicked back and restored to their previous state former to joining